### ISSUE TITLE
DragMap fails if --trim_fastq flag is enabled 

### ISSUE THREAD

**Issue by Ilaria-Pirona** [nf-core/sarek#1445]
### Description of the bug

Good evening,
I am experiencing problems with Sarek's pipeline. When I try to remove the adapters from my WES data, the dragmap tool runs into "Segmentation Fault". 
Do you have any idea what the problem might be?

Command line:
nextflow run main.nf
		 --input sample_sheet_3.csv
		 --genome GATK.GRCh38
		 --aligner dragmap
		 --save_output_as_bam
		 --step mapping
		 --outdir './results'
		 --save_output_as_bam
		 --wes false
		 -resume
		 -profile aws
		 --trim_fastq

**Comment by FriederikeHanssen** [comment]
What version are you running? Can you send the error message? there are several issues with dragmap that were never resolved: https://github.com/Illumina/DRAGMAP/issues/47

**Comment by edmundmiller** [comment]
Can you confirm it works without the `--trim_fastq` param?

**Comment by Ilaria-Pirona** [comment]
Yes I tried multiple times, and when i added --trim_fastq it failed.

Sarek by default put the arguments 
"--disable_adapter_trimming      --split_by_lines 200000000 \" in the process.

However, "--disable_adapter_trimming" to me is apparently in conflicts with "--detect_adapter_for_pe " which is written inside the module.

**Comment by FPGro** [comment]
This issue persists as of version 3.4.3. Related: https://github.com/Illumina/DRAGMAP/issues/11 

In my case, specifying any of `--trim_fastq` and/or `--trim_nextseq 10` results in dragmap segfaulting. I also tried `--three_prime_clip_r1 2` and `--three_prime_clip_r2 2` and these don't seem to cause issues.

Since dragmap does not seem to be actively maintained at the moment, but there is an issue in place to replace it with a better maintained fork [https://github.com/nf-core/sarek/pull/1358](https://github.com/nf-core/sarek/pull/1358), I suggest putting a warning into the parameter docstrings of `--aligner` that fastp is currently buggy with dragmap.

**Comment by nicorap** [comment]
+1 here. I run only alignment with dragmap and just haplotypecaller.  this is on version 3.7.1
